### DISCUSSION TITLE
Is the MQTT plugin and router currently supported on (implemented for) windows?

### DISCUSSION THREAD

**Discussion by andrew-otiv** [eclipse-zenoh/roadmap#96] [Q&A] [answered]
I ask because there are binary release for windows at https://download.eclipse.org/zenoh/zenoh-plugin-mqtt/latest/, but they're available for zenoh itself. 

Thanks!

**Comment by JEnoch** [comment, marked as answer]
We never tested it on Windows (neither ARM 32-bits). But since the the code is building fine for those platforms, I think it should work.

I activated those builds here: https://github.com/eclipse-zenoh/zenoh-plugin-mqtt/pull/20
We'll test and made those available for the next release.

In the meantime, you can test the binaries generated in nightly builds such as here (see "Artifacts" section at the bottom:
https://github.com/eclipse-zenoh/zenoh-plugin-mqtt/actions/runs/6458422379
Please, let us know if it works for you!
